extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Podcast Header Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+/**
+ * Template variables.
+ *
+ * @var string $player_id
+ * @var string $title
+ * @var string $link
+ * @var array  $track
+ * @var array  $primary_colors
+ */
+
+if ( ! isset( $title ) && empty( $track ) && ! isset( $track['title'] ) ) {
+	return;
+}
+?>
+
+<h2 id=<?php echo esc_attr( $player_id ); ?>__title" class="jetpack-podcast-player__title">
```

Comment:
The value of `id` needs to be wrapped in double quotes.

Reply:
Nice catch, I couldn't find where it was missing when I tested it yesterday

Reply:
Fixed in 0e82ba81dc24c89d92a82ee303190a3ecc67e4d0